src/components/CompleteProfile/CompleteProfile.css

```
@@ -0,0 +1,413 @@
+#complete-profile-form {
+	padding: var(--spacing-lg) var(--spacing-md);
+
+	h2, #submit-button, #fields-wrapper {
+		display: block;
+		margin-inline: auto;
+		width: fit-content;
+	}
+
+	#fields-wrapper {
+		margin-block: 2.22rem;
+		width: clamp(21.375rem, 100%, 40rem);
+
+		details {
+			border: var(--border);
+			border-color: var(--color-accent);
+			width: 100%;
+
+			&:first-of-type {
+				border-start-end-radius: var(--rounded-lg);
+				border-start-start-radius: var(--rounded-lg);
+				overflow: clip;
+			}
+
+			&:last-of-type {
+				border-end-end-radius: var(--rounded-lg);
+				border-end-start-radius: var(--rounded-lg);
+				overflow: clip;
+			}
+
+			summary {
+				align-items: center;
+				background-color: var(--color-accent);
+				color: var(--color-white);
+				display: flex;
+				font-weight: var(--weight-bold);
+				gap: var(--spacing-xs);
+				list-style: none;
+				padding: var(--spacing-xs) var(--spacing-sm);
+
+				&::after {
+					background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='24' height='24' viewBox='0 0 24 24' fill='none'%3E%3Cpath d='M6 9L12 15L18 9' stroke='white' stroke-width='2' stroke-linecap='round' stroke-linejoin='round'/%3E%3C/svg%3E");
+					background-position: center;
+					background-repeat: no-repeat;
+					background-size: contain;
+					content: '';
+					display: inline-block;
+					height: 1rem;
+					width: 1rem;
+				}
+
+				[open] &::after { transform: rotate(180deg); }
+
+				& > span { flex-grow: 1; }
+			}
+
+			.details-content-wrapper { padding: var(--spacing-sm); }
+
+			.details-nutshell-grid {
+				display: grid;
+				gap: var(--spacing-sm);
+				grid-template-columns: repeat(3, 1fr);
+				justify-content: space-between;
+
+				& > div {
+					display: flex;
+					flex-direction: column;
+					gap: var(--spacing-xs);
+				}
+			}
+
+			.dob-wrapper {
+				display: grid;
+				gap: 0.5rem 0;
+				grid-template-columns: 2fr 3fr;
+				justify-items: start;
+
+				& label { font-size: var(--text-sm); }
+
+				& select { width: 100%; }
+			}
+
+			/* Media query for smaller screens */
+			@media (max-width: 48rem) { .details-nutshell-grid { grid-template-columns: 1fr; } }
+
+			.input-required {
+				align-items: center;
+				display: flex;
+				justify-content: space-between;
+			}
+
+			.input-required::after {
+				color: var(--color-accent);
+				content: 'REQUIRED';
```

```suggestion
				content: 'Required' / '';
				text-transform: uppercase;
```

The second part of the `content` property is text added to accessibility technologies for generated content. Without it all modern browsers will announce the "required" text. 🥲

It is only recently supported however (since 2024), so for other projects add it with and without the alt text.

The `text-trasnform` property does auto transformation for us.

---

I didn't know this one. Thanks for sharing.